simapp/app_config.go

```diff
@@ -100,195 +100,197 @@ var (
 		// govtypes.ModuleName
 	}
 
-	// application configuration (used by depinject)
-	AppConfig = depinject.Configs(appconfig.Compose(&appv1alpha1.Config{
-		Modules: []*appv1alpha1.ModuleConfig{
-			{
-				Name: runtime.ModuleName,
-				Config: appconfig.WrapAny(&runtimev1alpha1.Module{
-					AppName: "SimApp",
-					// NOTE: upgrade module is required to be prioritized
-					PreBlockers: []string{
-						upgradetypes.ModuleName,
-					},
-					// During begin block slashing happens after distr.BeginBlocker so that
-					// there is nothing left over in the validator fee pool, so as to keep the
-					// CanWithdrawInvariant invariant.
-					// NOTE: staking module is required if HistoricalEntries param > 0
-					BeginBlockers: []string{
-						minttypes.ModuleName,
-						distrtypes.ModuleName,
-						protocolpooltypes.ModuleName,
-						slashingtypes.ModuleName,
-						evidencetypes.ModuleName,
-						stakingtypes.ModuleName,
-						authz.ModuleName,
-						epochstypes.ModuleName,
-					},
-					EndBlockers: []string{
-						govtypes.ModuleName,
-						stakingtypes.ModuleName,
-						feegrant.ModuleName,
-						group.ModuleName,
-						protocolpooltypes.ModuleName,
-					},
-					OverrideStoreKeys: []*runtimev1alpha1.StoreKeyConfig{
-						{
-							ModuleName: authtypes.ModuleName,
-							KvStoreKey: "acc",
-						},
+	ModuleConfig = []*appv1alpha1.ModuleConfig{
```

Comment:
factored this out so that we can do some modifications to it in the network tests